cdci_data_analysis/analysis/parameters.py

```diff
@@ -164,13 +164,29 @@ def __init__(self,
                  value=None,
                  units=None,
                  name: Union[str, None] = None,
+                 # TODO should we remove units/type/format knowledge from the Parameter class?
+"""
+format:
+   every parameter has a format. Format defines representation of the parameter value. 
+   Default format of  defines unique representation 
+  
+TODO
+"""
```

Comment:
I think this needs correct indentation

Reply:
yeah, the suggestion did not embed correctly. Please feed free to adapt @dsavchenko ? Also I expended it with another suggestion.

Reply:
Done

Reply:
See I expanded it.

Reply:
Hurried again. Sorry